src/emoji/emojiSelectors.js

```diff
@@ -42,8 +42,12 @@ export const getAllImageEmojiByCode: Selector<{|
   objectFromEntries(Object.keys(emojis).map(id => [emojis[id].code, emojis[id]])),
 );
 
-export const getActiveImageEmojiByName: Selector<{|
```

Comment:
emoji [nfc]: Remove getActiveImageEmojiByName.
    
    Removes unused selector getActiveImageEmojiByName.

nit: I like having one bit more information in the commit message for this sort of change: the fact that it's unused because we just in the previous commit stopped using it. That has a different feel when I'm reading it from when it's something we stopped using months or years ago, and just noticed (which also happens.)

Reply:
Fixed.